modules/swagger-codegen/src/main/resources/Javascript/es6/partial_model_generic.mustache

```diff
@@ -65,7 +65,7 @@ export default class {{classname}} {{#parent}}{{^parentModel}}{{#vendorExtension
     * @member {{=< >=}}{<&vendorExtensions.x-jsdoc-type>}<={{ }}=> {{baseName}}{{#defaultValue}}
     * @default {{{defaultValue}}}{{/defaultValue}}
     */{{/emitJSDoc}}
-    {{baseName}} = {{#defaultValue}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}undefined{{/defaultValue}};
+    {{name}} = {{#defaultValue}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}undefined{{/defaultValue}};
```

Comment:
@CodeNinjai I still think this should be {{name}} instead of {{baseName}}

e.g. {{baseName}} (original name specified in the spec) can be "modified-date" while {{name}} is the variable name following JS style guide (e.g. modified_date)

cc @cliffano @frol

Reply:
@wing328 So that what the change does, doesn't it? It changes `{{ baseName }}` to `{{ name }}`.

Reply:
@frol ah sorry. You're right. I misread.